apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/tests/TestSharding.java

```diff
@@ -0,0 +1,60 @@
+package com.apollocurrrency.aplwallet.inttest.tests;
+
+import com.apollocurrency.aplwallet.api.dto.ShardDTO;
+import com.apollocurrrency.aplwallet.inttest.helper.TestConfiguration;
+import com.apollocurrrency.aplwallet.inttest.model.TestBaseNew;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.bouncycastle.crypto.tls.MACAlgorithm;
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.*;
+import static org.junit.Assert.assertThat;
+
+public class TestSharding extends TestBaseNew {
+
+
+    @Test
+    void verifyShards(){
+     List<String> peers = TestConfiguration.getTestConfiguration().getPeers();
+        HashMap<String, List<ShardDTO>> shards = new HashMap<>();
+        HashMap<String, Integer> heights = new HashMap<>();
+        List<ShardDTO> maxShardsList = new ArrayList<>();
+        int maxHeight = 0;
+
+        for (String ip: peers) {
+            try {
+                List<ShardDTO> shardDTOS = getShards(ip);
+                if (shardDTOS.size() > maxShardsList.size()){maxShardsList =shardDTOS;}
+                shards.put(ip, shardDTOS);
+                heights.put(ip,getLastBlock(ip).getHeight());
+                System.out.println(getLastBlock(ip));
```

Comment:
Maybe use logger?

Reply:
Yes, it is temporary code. I will be delete it.